libcloudforensics/providers/gcp/internal/storage.py

```diff
@@ -239,3 +238,37 @@ def GetBucketSize(self,
             elif val > ret[bucket]:
               ret[bucket] = val
     return ret
+
+  def CreateBucket(
+      self,
+      bucket: str,
+      labels: Optional[Dict[str, str]] = None,
+      predefinedacl: str = 'private',
+      predefineddefaultobjectacl: str = 'private') -> Dict[str, Any]:
```

Comment:
```suggestion
      predefined_acl: str = 'private',
      predefined_default_object_acl: str = 'private') -> Dict[str, Any]:
```

Reply:
Please also adjust variables names in the function.

Reply:
Done